openapi.yaml

```diff
@@ -322,6 +322,39 @@ info:
         }'
     ```
 
+    # Rate Limiting
+
+    With the Linode API, you can make up to 1600 general API requests every two minutes per user as
+    determined by IP adddress or by OAuth token.
+
+    If a user has multiple OAuth tokens, the quota of requests per user is shared amongst all tokens
+    for the user.
```

Comment:
I don't think this is correct... our limiter `key_func` (located [here](https://bits.linode.com/LinodeAPI/apinext/blob/development/apinext/services/limiting/__init__.py#L17-L19)) takes an md5 hash of the bearer token and uses that to determine the "bucket in which each request is put into when evaluating whether it is within the rate limit or not" (from the [limiter docs](https://flask-limiter.readthedocs.io/en/stable/#rate-limit-domain)), so I think if a user has multiple OAuth token, the quota or requests per users is _unique_ for each token.